camayoc/tests/qcs/cli/test_scans.py

```diff
@@ -249,12 +386,37 @@ def test_clear(isolated_filesystem, qpc_server_config, source):
         1) Run ``qpc scan add --sources <source>``
         2) Run ``qpc scan clear --name <name>``
     :expectedresults: Scan is deleted.
-    :caseautomation: notautomated
     """
-    pass
+    # Create scan
+    scan_name = uuid4()
+    source_name = config_sources()[0]['name']
+    scan_add({
+        'name': scan_name,
+        'sources': source_name
+    })
+
+    source_output = source_show_output({'name': source_name})
+    source_output = json.loads(source_output)
+
+    expected_result = {'id': 'TO_BE_REPLACED',
+                       'name': scan_name,
+                       'options': {
+                           'max_concurrency': 50},
+                       'scan_type': 'inspect',
+                       'sources': [{'id': source_output['id'],
+                                    'name': source_name,
+                                    'source_type': 'network'}]}
+
+    scan_show(scan_name, expected_result)
+
+    # Remove scan
+    result = scan_clear({
+        'name': scan_name
+    })
+    match = re.match(r'Scan "{}" was removed.'.format(scan_name), result)
```

Comment:
I'm noticing your use of `r'string ...'`. Curious what came up that prompted making these "raw" strings. It would be good if you could add some comment somewhere explaining why you needed that.

Reply:
It is a good practice to create a raw string for regex so you don't need to escape `\`.

Reply:
Yep, what @elyezer  said.  Regex typically use raw.